src/allDrugs.jl

```diff
@@ -273,3 +273,19 @@ function blissCellNum(g1s, g2s; T = 96, n = 8)
     @assert(all(combined .>= 0.0))
     return combined
 end
+
+""" Function for calculating temporal combination of two drugs. """
+function temporal_combination(params1, params2, g0)
+    t1 = LinRange(0.0, 47.5, 96)
+    t2 = LinRange(48.0, 95.5, 96)
```

Comment:
i.e. t2 here should start at 0.0 and go to 95.5-48.0. That's also likely why you're getting really large numbers.